src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/CannotConfirm.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@page "/apprenticeships/{apprenticeshipId:?}/cannotconfirm"
```

Comment:
I don't think `{apprenticeshipId}` should be optional.  That probably comes from my original example, sorry to mislead.

Reply:
Agree, but I'm waiting for the HashedId so I can rework this